build.gradle

```diff
@@ -64,7 +64,7 @@ allprojects {
     }
 
     afterEvaluate {
-        if (project.name != "qa") {
```

Comment:
bwc isnt a project. Why does this need to change?

Reply:
my bad. will update it.

Reply:
I'm removing the condition here, to apply license and notice to sub-projects to fix the following error:
```
A problem occurred configuring project ‘:qa’.
> Must specify license and notice file for project :qa
```